Lacework/lacework_module/client/__init__.py

```diff
@@ -0,0 +1,51 @@
+import requests
+from requests.adapters import Retry
+from requests.auth import AuthBase
+from requests_ratelimiter import LimiterAdapter
+
+from lacework_module.client.auth import LaceworkAuthentication
+
+
+class ApiClient(requests.Session):
+    def __init__(self, auth: AuthBase, nb_retries: int = 5, ratelimit_per_hour: int = 480):
+        super().__init__()
+        self.auth = auth
```

Comment:
I suggest providing the base url to the client (instead to get it from the authentication class).
```suggestion
    def __init__(self, base_url: str, auth: AuthBase, nb_retries: int = 5, ratelimit_per_hour: int = 480):
        super().__init__()
        self.base_url = base_url
        self.auth = auth
```